CHANGELOG.md

```diff
@@ -1,4 +1,171 @@
-# Change Log
+﻿# Change Log
```

Comment:
Should i fix this while i am at it?

![image](https://user-images.githubusercontent.com/103522/89214943-5ae0ac00-d58d-11ea-94a2-013b9ef90fa8.png)

Change Log or Changelog ?

Should the title here be `MonoGame Change Log` ?

Anything else we want to add to this?

Reply:
Changelog